pandas/tests/indexes/datetimes/test_timezones.py

```diff
@@ -574,6 +578,38 @@ def test_dti_tz_localize_bdate_range(self):
         localized = dr.tz_localize(pytz.utc)
         tm.assert_index_equal(dr_utc, localized)
 
+    @pytest.mark.parametrize('tz', ['Europe/Warsaw', 'dateutil/Europe/Warsaw'])
+    @pytest.mark.parametrize('method, exp', [
+        ['shift', '2015-03-29 03:00:00'],
+        ['NaT', pd.NaT],
```

Comment:
do you have tests that exericse the assertion when you pass a ``nonexistent`` keyword that is invalid?